src/router.js

```diff
@@ -315,6 +315,15 @@ export default (store) => {
             title: 'Retries'
           },
           component: () => import(/* webpackChunkName: "retries" */ '@/views/Policies/Retries')
+        },
+        // timeouts
+        {
+          path: 'timeouts',
+          name: 'timeouts',
+          meta: {
+            title: 'Timeouts'
+          },
+          component: () => import(/* webpackChunkName: "retries" */ '@/views/Policies/Timeouts')
```

Comment:
can you change the chunk name to `timeouts` and also add the file extension to it. it should be '@/views/Policies/Timeouts.vue'

Reply:
Thanks, fixed!